### DISCUSSION TITLE
Trying to test the docker container v1.2 and getting HTTP 500

### DISCUSSION THREAD

**Discussion by tapantfspl** [Sahamati/rahasya#27] [Q&A]
Hi. I am trying to use the docker container v1.2 before integration with our TSP code modules. 

I am getting HTTP 500 presently with no message. 

Steps that I executed. 
a. Started docker container on 8080 and browsed to swagger-ui 
b. Created 2 key pairs using "/x25519/v1/generateKey" - named them KP1 and KP2
c. Used KP1's private key and KP2's public key as input to "/x25519/v1/getSharedKey"

Got HTTP 500 response with following body. 

{
  "timestamp": "2022-10-11T07:34:16.849+0000",
  "status": 500,
  "error": "Internal Server Error",
  "message": "No message available",
  "path": "/x25519/v1/getSharedKey"
}

Assumptions - the generated key pairs are in proper format - I am not doing any conversion / transformation etc on the keys provided. 

Our use case is that we want the docker container to be hosted privately and call it from our product (which is a .NET Core based product). 

Any help will be appreciated. 

Thanks 
Tapan Trivedi (Finacus Solutions).

**Comment by tapantfspl** [comment]
Further update:

My request json to getSharedKey was faulty. It would have been good if the setters had JsonProperty defined. It would have trapped incorrect request.

**Reply by gsasikumar** [reply]
Provide a sample we can try and see if we can address that clearly

**Comment by tapantfspl** [comment]
Hi ,

I have deployed the service on a linux machine now and have exposed it as
APIs. Its working fine for me now.

For the specific use case of docker - my deployment is on windows and with
.Net dev environment. I have concluded that on Windows this
particular docker container is an issue (not the container per say but
docker on windows is somehow not working out).

It's not a priority presently.

Re Tapan

--
*Tapan Trivedi*
